Dockerfile

```
@@ -0,0 +1,22 @@
+ARG DOCKER_BASE_IMAGE
+FROM $DOCKER_BASE_IMAGE
+ARG VCS_REF
+ARG BUILD_DATE
+LABEL \
+    maintainer="https://ocr-d.de/kontakt" \
+    org.label-schema.vcs-ref=$VCS_REF \
+    org.label-schema.vcs-url="https://github.com/qurator-spk/sbb_binarization" \
+    org.label-schema.build-date=$BUILD_DATE
+
+WORKDIR /build/sbb_binarization
+COPY setup.py .
+COPY ocrd-tool.json .
```

```suggestion
COPY sbb_binarize/ocrd-tool.json .
```
Not sure if that is really needed though, but we do this in every other Dockerfile as well

---

Not needed. The root ocrd-tool.json is a symlink that resolves correctly in the build context, so it has the same effect. (But generally, writing Dockerfiles for OCR-D modules, you have to be much more careful with your form, because the target of the symlink might be some other location deeper inside the package.)